### PR TITLE
Change one sentence that was really hard to parse.

### PR DESCRIPTION
**Pull request by bruce-usab**
I had to read this sentence four times to figure subject / object / verb.  The fix turned to be pretty simple.

### CODE REVIEW

**Comment by jspellman** [comment]
Thank you for your comment. We have closed this issue because we believe this issue has been addressed or is no longer relevant. If you believe this issue should be reopened, please open a new issue and reference this issue.

**Review by michael-n-cooper — APPROVED** [review]

**Comment by michael-n-cooper** [comment]
Appears to be OBE
